site/content/en/docs/drivers/podman.md

```diff
@@ -11,7 +11,14 @@ aliases:
 This driver is experimental and in active development. Help wanted!
 {{% /pageinfo %}}
 
-The podman driver is another kubernetes in container driver for minikube. similar to [docker](https://minikube.sigs.k8s.io/Drivers/docker/) driver. The podman driver is  experimental, and only supported on Linux and macOS (with a remote podman server)
+The podman driver is another kubernetes in container driver for minikube. similar to [docker](https://minikube.sigs.k8s.io/Drivers/docker/) driver. The podman driver is  experimental, and only supported on Linux and macOS (with a remote podman server).
+
+
+
+## Requirements
+- amd64 system 
+- Latest version of [Podman](https://podman.io/getting-started/installation) 
```

Comment:
Shouldn't be required with latest version, then we will shut out platforms such as RHEL.

Probably should just say something like "Install Podman", similar to what Docker says ?